geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/IngestCommand.scala

```diff
@@ -26,7 +26,11 @@ class IngestCommand(parent: JCommander) extends Command(parent) with Logging {
   override def execute(): Unit = {
     val fmt = Option(params.format).getOrElse(getFileExtension(params.files(0)))
     fmt match {
-      case CSV | TSV => new DelimitedIngest(params).run()
+      case CSV | TSV =>
+        if (Option(params.spec).isEmpty) {
```

Comment:
Not necessary to create an option here if you're just checking it for empty: perfectly fine to do <code>if (params.spec == null)</code>